app/components/tag/tag_component.html.erb

```diff
@@ -0,0 +1,3 @@
+<strong class="govuk-tag <%= css_classes %>">
```

Comment:
For Apply we need to add a custom CSS class. Do we skip doing that for now?

Reply:
I think this would be worth while. On Publish we are using custom classes on tags

- https://github.com/DFE-Digital/publish-teacher-training/blob/master/app/decorators/application_decorator.rb
- https://github.com/DFE-Digital/publish-teacher-training/blob/master/app/views/providers/allocations/_request_status.html.erb#L23-L25

Reply:
Could follow a similar pattern to this https://github.com/DFE-Digital/govuk-components/blob/master/app/components/govuk_component/back_link.rb#L7-L8